### ISSUE TITLE
Potential problem with explode tool

### ISSUE THREAD

**Issue by pfuntner** [pfuntner/toys#92]
I did the following:
1. `explode -x aws-artifacts.zip gcp-artifacts.zip`
2. `aws-artifacts.zip` was unwound properly
3. `gcp-artifacts.zip` was not unwound properly
    1. There was a nastygram:
        ```
        2022-01-14 07:50:37,175 WARNING /home/jpfuntne/bin/explode:111 Error unwinding gcp-artifacts.zip to /home/jpfuntne/tmp/20220114072928971999600/gcp-artifacts:
          Archive:  /home/jpfuntne/tmp/20220114072928971999600/aws-artifacts.zip
          caution: filename not matched:  -xvf
          caution: filename not matched:  /home/jpfuntne/tmp/20220114072928971999600/gcp-artifacts.zip
        $
        ```
    2. The `gcp-artifacts` directory was created but it was empty
4. I was able to do `explode -x gcp-artifacts.zip` and unwind the zip fine after first deleting the `gcp-artifacts` directory

There definitely seems to be a problem but I don't know what it is.  The help is just:
```
$ explode
usage: explode [-h] (-l | -c | -x) [-z | -t | -g | -j] [-v] [file [file ...]]
explode: error: one of the arguments -l/--list -c/--count -x/--extract is required
$
```
I think I was using the command properly - I believe the file names are supposed to be independent and I wasn't telling it _extract `gcp-artifacts.zip` from `aws-artifacts.zip`_.  If that's the case, I need to sit down and figure out how to treat the arguments because I think a case could be made for it working either way.  Today, I think they should be independent but maybe tomorrow when I'm facing a different use case, I might have another take.

**Comment by pfuntner** [comment]
Verified... as long as I'm using it properly.

```
$ date > file1
$ date > file2
$ zip zip1 file1
  adding: file1 (stored 0%)
$ zip zip2 file2
  adding: file2 (stored 0%)
$ explode -x zip1.zip zip2.zip
Unwound zip1.zip to /media/mrbruno/ExtraDrive1/tmp/20220813082108812075557/zip1:
  Archive:  /media/mrbruno/ExtraDrive1/tmp/20220813082108812075557/zip1.zip
   extracting: file1                   

2022-08-13 08:21:57,751 WARNING /home/mrbruno/bin/explode:111 Error unwinding zip2.zip to /media/mrbruno/ExtraDrive1/tmp/20220813082108812075557/zip2:
  Archive:  /media/mrbruno/ExtraDrive1/tmp/20220813082108812075557/zip1.zip
  caution: filename not matched:  -xvf
  caution: filename not matched:  /media/mrbruno/ExtraDrive1/tmp/20220813082108812075557/zip2.zip

$ 
```

I think it's supposed to accept multiple archives on the command line - you're not specifying files **in** the previous archive.

**Comment by pfuntner** [comment]
Fixed with f35f05b
